osu.Game.Rulesets.Osu/Mods/OsuModRepel.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Utils;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Rulesets.Osu.UI;
+using osu.Game.Rulesets.Osu.Utils;
+using osu.Game.Rulesets.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModRepel : Mod, IUpdatableByPlayfield, IApplicableToDrawableRuleset<OsuHitObject>
+    {
+        public override string Name => "Repel";
+        public override string Acronym => "RP";
+        public override ModType Type => ModType.Fun;
+        public override string Description => "Hit objects run away!";
+        public override double ScoreMultiplier => 1;
+        public override Type[] IncompatibleMods => new[] { typeof(OsuModAutopilot), typeof(OsuModWiggle), typeof(OsuModTransform), typeof(ModAutoplay), typeof(OsuModMagnetised) };
```

Comment:
This mod is declaring incompatibility with 5 others, but 4 of those 5 are not declaring incompatibility with `OsuModRepel` themselves in the other direction (`OsuModMagnetised` being the exception).

Reply:
You still missed autoplay..

Reply:
and relax.

Reply:
Yikes.

Relax was intentionally excluded though, #17319 isn't an issue when hitobjects are being pushed away.